src/main/resources/db/migrations/V0_0_1__initialize.sql

```diff
@@ -0,0 +1,61 @@
+-- phpMyAdmin SQL Dump
+-- version 4.8.5
+-- https://www.phpmyadmin.net/
+--
+-- Host: mariadb
+-- Generation Time: Jun 02, 2019 at 12:32 PM
+-- Server version: 10.3.15-MariaDB-1:10.3.15+maria~bionic
+-- PHP Version: 7.2.14
+
+SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
```

Comment:
1.) Remove this SET value
2.) Split flyway into two version
3.) Remove Collate
4.) Remove database engine